incubator/group/testdata/sim_app.go

```diff
@@ -2,50 +2,64 @@ package testdata
 
```

Comment:
I wish there were a way we could just extend `simapp` from the sdk rather than just having everything copied here. Not relevant for this PR, but something to think about in the future.